cli/commands/update-cli.go

```diff
@@ -91,7 +91,7 @@ func (o *updateOptions) run(showPrompt bool) (err error) {
 				versionToBeInstalled = latestVersion
 			case hasPreReleaseUpdate:
 				o.EC.Logger.Infof(`a new pre-release version is available:
-- %s (changelog: %s)
+- %s (changelog: \u200B%s\u200B)
```

Comment:
```
o.EC.Logger.Infof("a new pre-release version is available:\n- %s (changelog: %s)", preReleaseVersion.Original(), getChangeLogLink(preReleaseVersion))
o.EC.Logger.Infof(`to update cli to this version, execute:

  hasura update-cli --version %s

`, preReleaseVersion.Original())
```

Reply:
Yup, that will work too. Your call.

Reply:
Issue is with change log URL so how would you suggest to be done. Adding spaces will also work.